software/script/chameleon_cmd.py

```diff
@@ -267,7 +318,7 @@ def write_em_410x_to_t55xx(self, id_bytes: bytearray):
             data.extend(key)
         return self.device.send_cmd_sync(DATA_CMD_WRITE_EM410X_TO_T5577, 0x00, data)
 
-    def set_slot_activated(self, slot_index):
+    def set_slot_activated(self, slot_index: SlotNumber):
```

Comment:
each such function is defined twice, make sure to fix the second one too (e.g. set_slot_activated is also at line 553).

Reply:
Those shouldn't need to be changed since they call but don't actually do data validation like the actual function itself.